<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JunctionPlacement</class>
 <widget class="QDialog" name="JunctionPlacement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_name">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_alias">
          <property name="text">
           <string>Alias</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_sysid">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_uid">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" rowspan="2" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_cattype">
          <property name="text">
           <string>Catalog type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_phasecode">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Phase Code</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_sysid">
          <property name="text">
           <string>SystemID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_uid">
          <property name="text">
           <string>UID</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="comboBox_cattype">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="lineEdit_name"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_alias"/>
        </item>
       </layout>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="comboBox_phasecode"/>
      </item>
      <item row="4" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_descr">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_decr"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0" alignment="Qt::AlignLeft">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>buttonBox</zorder>
  <zorder>buttonBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
